interview-questions.md

```diff
@@ -7,53 +7,71 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are (most) Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: In Learn bootcamp when we have Ruby methods class there we discussed about the ruby methods most of ruby methods are accessors 
 
-  Researched answer:
+  Researched answer: When I researched I learned that ruby methods are all accessors 
+  means they do not mutate objects they called on and for method to permanently change the object called on using bang operator
+ exp: num =[1,2,3,5]
+ num.reverse => [5,3,2,1]
+ num => [1,2,3,5]
+ num.reverse! => [5,3,2,1] used bang operator'!'
+ num => [5,3,2,1] permanantly changed
```

Comment:
Okay this answer is good but how would you express syntax like this in a face to face interview? We need to be ready to discuss this without syntax (or as little as possible) for when we are interviewing.

Reply:
yeah, I will try to dicuss more with out syntax..thank you